nodes/vector/turbulence.py

```diff
@@ -124,21 +123,12 @@ def process(self):
         m_freq = inputs['Frequency'].sv_get()[0]
         m_seed = inputs['Random seed'].sv_get()[0]
 
-        #if verts and verts[0]:
-
         maxlen = len(verts[0])
```

Comment:
no.. you'll always be able to do `len(verts)` . All verts data is (by convention) wrapped .
```
verts = [one_circle_list]
# len(verts) == 1
verts = [one_circle_list, two_circle_list]
# len(verts) == 2
verts = [one_circle_list, two_circle_list, three_circle_list]
# len(verts) == 3
```
test with circle node :)